### PR TITLE
Update Funding.yml, and PULL_REQUEST_TEMPLATE.md

### PR DESCRIPTION
**Pull request by skanda890**
- [X] **I have read the [contributing guidelines](https://github.com/marticliment/WingetUI/blob/main/CONTRIBUTING.md#coding), and I agree with the [Code of Conduct](https://github.com/marticliment/WingetUI/blob/main/CODE_OF_CONDUCT.md)**.
- [X] **Have you checked that there aren't other open [pull requests](https://github.com/marticliment/WingetUI/pulls) for the same changes?**
- [X] **Have you tested that the committed code can be executed without errors?**
- [X] **This PR is not composed of garbage changes used to farm GitHub activity to enter potential Crypto AirDrops.**
Any user suspected of farming GitHub activity with crypto purposes will get banned. Submitting broken code wastes the contributors' time, who have to spend their free time reviewing, fixing, and testing code that does not even compile breaks other features, or does not introduce any useful changes. Thank you for your understanding.

-----

Update Funding.yml, and PULL_REQUEST_TEMPLATE.md
-----

### CODE REVIEW

**Comment by skanda890** [comment]
@marticliment, please review this.

File: .github/FUNDING.yml [hunk 1]
```diff
@@ -1,13 +1,12 @@
 # These are supported funding model platforms
-
 github: # Replace with up to 4 GitHub Sponsors-enabled usernames e.g., [user1, user2]
 patreon: # Replace with a single Patreon username
 open_collective: # Replace with a single Open Collective username
 ko_fi: martinet101
 tidelift: # Replace with a single Tidelift platform-name/package-name e.g., npm/babel
-community_bridge: # Replace with a single Community Bridge project-name e.g., cloud-foundry
+community_bridge: # Replace with a single Community Bridge project name e.g., cloud-foundry
```
Comment (marticliment) [inline .github/FUNDING.yml]: This is documentation GitHub adds to funding.yml by default. I reckon if it has the dash, it must be for something. What do you think?

**Comment by marticliment** [comment]
please check the changed you proposed, the dash is not where it was

**Comment by skanda890** [comment]
> please check the changed you proposed, the dash is not where it was

Change done.

**Pull request merged**